Makefile

```diff
@@ -12,14 +12,15 @@ build-reqs:
 
 cython: build-reqs
 	find edb -name '*.pyx' | xargs touch
-	BUILD_EXT_MODE=py-only python setup.py build_ext --inplace
+	# See also: https://github.com/cython/cython/issues/5240
+	CFLAGS="${CFLAGS} -Wno-error=incompatible-pointer-types" BUILD_EXT_MODE=py-only python setup.py build_ext --inplace
```

Comment:
Add these to `setup.py` `CFLAGS` instead.

Reply:
Ah makes sense!